rules/legacy/tests/Rector/Include_/AddTopIncludeRector/AddTopIncludeRectorTest.php

```diff
@@ -4,33 +4,47 @@
 
 namespace Rector\Legacy\Tests\Rector\Include_\AddTopIncludeRector;
 
-use Iterator;
 use Rector\Core\Testing\PHPUnit\AbstractFileSystemRectorTestCase;
 use Rector\Legacy\Rector\Include_\AddTopIncludeRector;
 use Symplify\SmartFileSystem\SmartFileInfo;
 
 final class AddTopIncludeRectorTest extends AbstractFileSystemRectorTestCase
 {
-    /**
-     * @dataProvider provideData()
-     */
-    public function test(SmartFileInfo $fileInfo): void
+    public function test(): void
+    {
+        $fileInfo = new SmartFileInfo(__DIR__ . '/Fixture/fixture.php.inc');
+        $temporaryFileInfo = $this->doTestFileInfo($fileInfo);
+
+        $this->assertStringEqualsFile(__DIR__ . '/Expected/expected_autoload.php', $temporaryFileInfo->getContents());
+    }
```

Comment:
File system is a bit more complicated than normal tests, I've update it here so the file is tested.